README.md

```diff
@@ -44,11 +44,94 @@ The object has the following schema (validated [here](./lib/index.js) using [Joi
   * `done: (err: Error, key: String)` - A function that takes an error as the first parameter and the bucket key as the second parameter.
 
 **Optional**
+* `enabled: Boolean default(true)` - if `true` rate limiting will be enabled you can set it to `false` on route settings to disable rate limiting
+* `addHeaders: Boolean default(true)` - if `true` rate limiting headers will be sent with response
+* `count: Number default(1)` - how many tokens request should cost useful when you want one route to be more expensive than another but use the same bucket
```

Comment:
Just thought it might be useful as a route option.
Personally I don't have a use case for it